WordPressKit/ReaderPostServiceRemote+Cards.swift

```diff
@@ -0,0 +1,34 @@
+extension ReaderPostServiceRemote {
+    func fetchCards(for interests: [String],
+                    success: @escaping ([RemoteReaderCard]) -> Void,
+                    failure: @escaping (Error) -> Void) {
+        guard let requestUrl = cardsEndpoint(for: interests) else {
+            return
+        }
+
+        wordPressComRestApi.GET(requestUrl,
+                                parameters: nil,
+                                success: { (response, _) in
+
+                                    do {
+                                        let decoder = JSONDecoder()
+                                        let data = try JSONSerialization.data(withJSONObject: response, options: [])
+                                        let envelope = try decoder.decode(ReaderCardEnvelope.self, from: data)
+
+                                        success(envelope.cards)
+                                    } catch {
+                                        DDLogError("\(error)")
```

Comment:
This will log not only the error message but also the file, function, and line: https://github.com/wordpress-mobile/WordPressKit-iOS/blob/develop/WordPressKit/CocoaLumberjack.swift#L86

In that case, I don't think we need to be descriptive in the error message. Wdyt?

Reply:
I think adding a small description of what the code was trying to do when it error'd out is important. This is also pretty common in other aspects of the app that use `DDLogError`.